src/modules/dataman/dataman.cpp

```diff
@@ -1432,6 +1434,8 @@ task_main(int argc, char *argv[])
 	px4_sem_destroy(&g_work_queued_sema);
 	px4_sem_destroy(&g_sys_state_mutex_mission);
 	px4_sem_destroy(&g_sys_state_mutex_fence);
+	perf_free(_dm_read_perf);
+	perf_free(_dm_write_perf);
```

Comment:
Can you add `_dm_write_perf = nullptr;`?

Reply:
Done.